packages/documentation/src/stories/components/accordion/accordion.stories.ts

```diff
@@ -99,39 +100,38 @@ export const DefaultCollapsedPanels: Story = {
   `,
 };
 
-// Used only in testing for now.
 export const Nested: Story = {
   render: () => html`
-    <post-accordion>
+    <post-accordion heading-level="4">
       <post-accordion-item>
-        <span slot="header">Titulum 1</span>
+        <span slot="header">Titulum 1 <code>h4</code></span>
         <p>Contentus momentus vero siteos et accusam iretea et justo.</p>
 
-        <post-accordion>
+        <post-accordion heading-level="5">
```

Comment:
If you use the storybook logic to override args on the story level, you could change this line to:
`<post-accordion heading-level=${Math.min(args.headingLevel + 1, 6)}>`